fishjam/server_notifications.proto

```diff
@@ -153,6 +153,28 @@ message ServerMessage {
     Track track = 4;
   }
 
+  // Notification sent when streamer successfully connects
+  message StreamConnected {
+    string stream_id = 1;
```

Comment:
Those notifications will be used for both the broadcaster and server SDKs. Did you decide what you will use? (room, live stream)